hw/top_earlgrey/data/ip/chip_i2c_testplan.hjson

```diff
@@ -0,0 +1,94 @@
+            - The SW test writes a known payload over the chip's I2C host interface, which is
+              received by the testbench.
+            - The testbench then loops this data back to the chip's I2C host and exercises the
+              read interface.
+            - SW validates the reception of FMT watermark and trans complete interrupts.
+            - SW validates that the data read matches the original data written.
+            - Verify the virtual / true open drain capability.
+
+            Verify all instances of I2C in the chip.
+
+            Notes for silicon targets:
+            - The testbench in this case will be a device that physically toggles/inspects the appropriate SCL and SDA lines.
+            '''
+      features: ["I2C.MODE.HOST", "I2C.OPERATION.READ", "I2C.OPERATION.WRITE"]
+      stage: V2
+      si_stage: SV2
+      tests: ["chip_sw_i2c_host_tx_rx",
+              "chip_sw_i2c_host_tx_rx_idx1",
+              "chip_sw_i2c_host_tx_rx_idx2"]
+    }
+    {
+      name: chip_sw_i2c_device_tx_rx
+      desc: '''Verify the transmission of data over the chip's I2C device interface.
+
+            - Program the I2C to be in device mode.
+            - The testbench writes a known payload over the chip's I2C device interface, which is
+              received and verified by the SW test for correctness.
+            - The testbench reads and verifies a known payload over the chip's I2C device interface,
+            - SW validates the reception of tx empty and trans complete interrupts.
+            - Verify the virtual / true open drain capability.
+
+            Verify all instances of I2C in the chip.
+
+            Notes for silicon targets:
+            - The testbench in this case will be a device that physically toggles/inspects the appropriate SCL and SDA lines.
+            '''
+      features: ["I2C.MODE.TARGET", "I2C.OPERATION.READ", "I2C.OPERATION.WRITE"]
+      stage: V2
+      si_stage: SV2
+      tests: ["chip_sw_i2c_device_tx_rx"]
+    }
+    {
+      name: chip_sw_i2c_speed
+      desc: '''Verify the transmission speed of data over the chip's I2C interfaces.
+
+            Run the following for standard, fast and fast plus speeds:
+            - Program the I2C to be in the appropriate mode (host/target).
+            - For hosts, the SW test writes a known payload over the chip's I2C host interface.
+              The test checks whether the payload is sent at the appropriate baud-rate
+            - For targets, the test sends data at the appropriate baud-rate and checks whether an ack is received.
+
+            Verify all instances of I2C in the chip.
+
+            Notes for silicon targets:
+            - The testbench in this case will be a device that physically toggles/inspects the appropriate SCL and SDA lines.
+            '''
+      features: ["I2C.MODE.HOST", "I2C.MODE.TARGET", "I2C.SPEED.STANDARD", "I2C.SPEED.FAST", "I2C.SPEED.FASTPLUS"]
+      stage: V2
```

Comment:
nit: `stage: V3` to avoid lowering the current regression completion results. Same for `chip_sw_i2c_override`.